.github/ISSUE_TEMPLATE/QUESTION.md

```diff
@@ -0,0 +1,18 @@
+---
+name: ❓ Support Question
```

Comment:
TBD: Need to agree on (potential) emoji and the name

(this is what is used in [Swift Crypto](https://github.com/apple/swift-crypto/blob/master/.github/ISSUE_TEMPLATE/QUESTION.md))